tests/php/test-amp-style-sanitizer.php

```diff
@@ -2409,4 +2413,255 @@ public function test_prioritized_stylesheets( $html_generator, $assert ) {
 
 		$assert( $original_dom, $html, $amphtml_dom, $amphtml_dom->saveHTML(), $sanitizer->get_stylesheets() );
 	}
+
+	/**
+	 * Provide data to test_get_stylesheet_priority().
+	 *
+	 * @return array[] Array of test data.
+	 */
+	public function get_get_stylesheet_priority_data() {
+		return [
+			// Non-AMP handle.
+			[ [ 'link', [ 'id' => 'mediaelement-css' ] ], 1000 ],
```

Comment:
Note: It would be better if the comment "Non-AMP handle" were made into the array key, as this would then make it easier to identify the dataset when failures occur.

For example:

```suggestion
			'Non-AMP handle' => [ [ 'link', [ 'id' => 'mediaelement-css' ] ], 1000 ],
```